### ISSUE TITLE
Statusbar color space

### ISSUE THREAD

**Issue by hoaxdream** [siduck/chadwm#76]
Hi again sir. how did you make your status2d with spacing and not let the color fill the bar?

mine look like this:
![image1](https://i.imgur.com/E8TRPyc.png)

but yours have spacing in top and bottom and the colors doesnt fill the bar:
![image2](https://i.imgur.com/8YA0CJj.png)

**Comment by siduck** [comment]
u mean the padding around the statusqd colors?

**Comment by hoaxdream** [comment]
yes sir, when i try to add colors using status2d, it fills the whole date colors, unline yours it has spacing around the colors.

**Comment by siduck** [comment]
oh why did you close this issue?

**Comment by hoaxdream** [comment]
yea i gave up on this one. cant do it lol thanks anyway

**Comment by siduck** [comment]
maybe https://www.reddit.com/r/suckless/comments/on80gv/dwm_statuspadding_patch_conflicts_with_status2d/

**Comment by hoaxdream** [comment]
i got it working, but got one problem that conflict with my other patch. thanks again bro
